testdata/event_listener

```diff
@@ -244,11 +244,89 @@ zmemtbl         0     0 B
  titers         0
  filter         -       -    0.0%  (score == utility)
 
+# Set up a scenario where the table to be ingested overlaps with the memtable.
+# The table is ingested as a flushable. The flush metrics refect the flushed
+# ingestion.
+
+ingest-flushable
+----
+sync-data: wal/000012.log
+link: ext/1 -> db/000017.sst
+[JOB 13] ingesting: sstable created 000017
+sync: db
+sync-data: wal/000012.log
+close: wal/000012.log
+reuseForWrite: wal/000007.log -> wal/000018.log
+sync: wal
+[JOB 14] WAL created 000018 (recycled 000007)
+sync-data: wal/000018.log
+sync-data: wal/000018.log
+close: wal/000018.log
+create: wal/000019.log
+sync: wal
+[JOB 15] WAL created 000019
+remove: ext/1
+[JOB 13] ingested as flushable 000017 (826 B)
+sync-data: wal/000019.log
+close: wal/000019.log
+create: wal/000020.log
+sync: wal
+[JOB 16] WAL created 000020
+[JOB 17] flushing 1 memtable to L0
+create: db/000021.sst
+[JOB 17] flushing: sstable created 000021
+sync-data: db/000021.sst
+close: db/000021.sst
+sync: db
+sync: db/MANIFEST-000016
+[JOB 17] flushed 1 memtable to L0 [000021] (770 B), in 1.0s (2.0s total), output rate 770 B/s
+[JOB 18] flushing ingested tables
+create: db/MANIFEST-000022
+close: db/MANIFEST-000016
+sync: db/MANIFEST-000022
+create: db/marker.manifest.000007.MANIFEST-000022
+close: db/marker.manifest.000007.MANIFEST-000022
+remove: db/marker.manifest.000006.MANIFEST-000016
+sync: db
+[JOB 18] MANIFEST created 000022
+[JOB 18] flushed 1 ingested flushable L0:000017 (826 B) in 1.0s (2.0s total), output rate 826 B/s
+remove: db/MANIFEST-000014
+[JOB 18] MANIFEST deleted 000014
+[JOB 19] flushing 1 memtable to L0
+sync: db/MANIFEST-000022
+[JOB 19] flush error: pebble: empty table
```

Comment:
#2388 should solve for these errors. We can follow up once a fix for that lands.